docs/config.json

```diff
@@ -326,9 +326,13 @@
           "slug": "/application-access/guides/",
           "entries": [
             {
-              "title": "Connecting Apps",
+              "title": "Web Apps Access",
```

Comment:
```suggestion
              "title": "Web App Access",
```
Same as, for example, "Database Management System" or "Static Site Generator"—the final verb acts on many instances of the initial noun, but we tend to make the noun singular rather than plural.